pyworkflow/em/convert.py

```diff
@@ -28,8 +28,14 @@
 import sys
 from itertools import izip
 import PIL
+try:
```

Comment:
This will abort everything and will not let Scipion open if xmipp not available. I will vote for having a "fake xmipp"  module that allows Scipion to work under minimal conditions.